README.md

```diff
@@ -105,6 +103,56 @@ React.render(<App person={person} />, document.body)
 person.name = "Mike" // will cause the Observer region to re-render
 ```
 
+In case you are a fan of render props, you can use that instead of children. Be advised, that you cannot use both approaches at once, children have a precedence.
+
+Here goes example showing render prop only
```

Comment:
This line is a bogus one, I did not mean to include it by using _italic text_ :)